guides/source/getting_started.md

```diff
@@ -153,6 +153,16 @@ To install Rails, use the `gem install` command provided by RubyGems:
 ```bash
 $ gem install rails
 ```
+TIP: If you are using an Arch/Arch based Linux system and have installed Ruby using pacman,
+gems like `rails` won't automatically be added to your path and will not be accessible from the command line.
+To fix this, add the lines:
```

Comment:
```suggestion
To fix this, add following lines to your `.bashrc` file and run `source $HOME/.bashrc` inside your terminal. The `rails` command should now work correctly.
```

Reply:
Yes, I hadn't thought about that and and agree with all your points. It sounds like the much better option to be honest and really isn't Rails related.